src/main/kotlin/io/github/loxygen/aimlessbot/cmds/tests/Ping.kt

```diff
@@ -0,0 +1,34 @@
+package io.github.loxygen.aimlessbot.cmds.tests
+
+import io.github.loxygen.aimlessbot.lib.commands.CommandResult
+import io.github.loxygen.aimlessbot.lib.commands.abc.PrefixnessCommandExecutor
+import io.github.loxygen.aimlessbot.lib.commands.annotations.Argument
+import io.github.loxygen.aimlessbot.lib.commands.annotations.SubCommand
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent
+import kotlin.random.Random
+
+object Ping : PrefixnessCommandExecutor(
+   identify = "ping",
+   name = "Ping",
+   description = "フェイクのping値を返します。"
+) {
+
+   @SubCommand(identify = "exec", name = "実行", description = "引数をつけるとPing値がその値になります")
+   @Argument(count = 1, denyLess = false)
```

Comment:
denyLess、requiredとかに名前変えませんか
```suggestion
   @Argument(count = 1, required = false)
```

Reply:
あ、ガバじゃんこれ
引数の数が少ないことを許容するかどうかか